README.md

```diff
@@ -1 +1,136 @@
+- [Key feature](#Key feature)
+ -[🚀 Live Demo](#live-demo)
+- [Setup](#setup)
+- [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+# JavaScript-Capstone-Group-Project-TV-API
+The JavaScript capstone project is about building our web application based on an external API. We've selected a TV-based API and then build our web app around it. The web app has 2 user interfaces.
+
+## Built With
+HTML
+CSS
+JavaScript (ES6)
+Linters
+React
+
+## Key feature
+Here are some key features of the application:
+Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+-[] Make simple calculations.
+-[] Read a random math-related quote.
+
+## Important Links
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](Coming soon)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+To get a local copy up and running follow these simple example steps.
+
+### PrerequisitesTo run this project you need:
+
+You should have the node installed on your local machine
+
+### Setup
+
+```
+git clone https://github.com/elad237/Math-Magician.git
+I run this command
+npx create-react-app my-app  to install react, react-dom, and react-scripts with cra-template...
+```
+
+### Install
+
+- Install all the dependencies required for the website. By using the command below.
+```
+Run npm install to install all dev dependencies for Linters, Webpack, Jest & Babel.
+```
+  
+- Open the index.html file in a browser.
+
+### Usage
+
+To run the project, execute the following command:
+
+Run npm run start in your command terminal.
+
+### Run tests
+
+o run tests, run the following command:
+
+Example command:
+npm install --save-dev jest to install jest
+npm run test to run the test
+npm run start to start your application using webpack
+To run the webhint linter
+
+  npx hint .
+To run the stylehint linter
+
+  npx stylehint "**/*.{css,scss}"
+To run the ESLint linter
+
+  npx eslint .
+### Deployment
+You can deploy this project using:
+
+npm start
+    Starts the development server.
+
+  npm run build
+    Bundles the app into static files for production.
+
+  npm test
+    Starts the test runner.
+
+  npm run eject
+    Removes this tool and copies build dependencies, configuration files
+    and scripts into the app directory. If you do this, you can’t go back!
+
+We suggest that you begin by typing:
+
+  cd my-app
+  npm start
+## Author 
+
+👤 **TUMBU BARTHOLOMEW ELAD**
+GitHub: [@githubhandle](https://github.com/elad237)
+- Twitter: [@twitterhandle](https://twitter.com/Elad59380989)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tumbu-elad-896ab2183/)
+
+#future-features
```

Comment:
- [ ] OPTIONAL
Please ensure that all headings stand out correctly so as to make your ReadMe appear more professional!